### ISSUE TITLE
Datum plausibilisieren

### ISSUE THREAD

**Issue by rradczewski** [socramob/Agora#32]
Wir sollten mindestens 0 Values (also 1.1.1970) noch zusätzlich rausfiltern.

https://github.com/rradczewski/Agora/blob/develop/lib/activityresults/index.js#L71

**Comment by andreassimon** [comment]
ja. wobei ich überlege, ob 0 nicht sowieso falsy ist und somit schon rausgefiltert wird.

**Comment by leider** [comment]
Meine Fotos waren direkt aufgenommen, warum nehmt ihr nicht das datum des hochladens?

**Comment by andreassimon** [comment]
tun wir seit gestern abend: https://github.com/socramob/Agora/blob/master/lib/activityresults/index.js#L71-74
